pkg/framework/plugins/nodeutilization/usageclients.go

```diff
@@ -100,3 +103,84 @@ func (s *requestedUsageClient) sync(nodes []*v1.Node) error {
 
 	return nil
 }
+
+type actualUsageClient struct {
+	resourceNames         []v1.ResourceName
+	getPodsAssignedToNode podutil.GetPodsAssignedToNodeFunc
+	metricsCollector      *metricscollector.MetricsCollector
+
+	_pods            map[string][]*v1.Pod
```

Comment:
do we need `_` already not exported anyways?

Reply:
`actualUsageClient` already defines `pods` as a method. Prefixing a private variable with `_` is a common practice. Yet, `internalPodList` or similar would do as well.